data/scoreboard/__init__.py

```diff
@@ -41,6 +41,12 @@ def strikeout(self):
 
     def strikeout_looking(self):
         return self.play_result == "strikeout_looking"
+    
+    def hit(self):
+        return self.play_result == "single" or self.play_result == "double" or self.play_result == "triple"
```

Comment:
```suggestion
        return self.play_result in ["single", "double", "triple"]
```

The other comment I made constantizes the plays, I'd really like to do just this, depending on how it needs to get imported

```python
        return self.play_result in data.plays.HITS
```